### DISCUSSION TITLE
Laser pointers detection issue

### DISCUSSION THREAD

**Discussion by Izh003** [biigle/core#766] [General]
Hi, I have created two volumes with images. I am having trouble identifying laser points (thus, image area and total area) in both of the volumes.
In the first one, I get a message "The laser point detection was submitted and will be available soon" after I compute the area of each image in this volume. The laser pointers are detected for a limited amount of images only. I annotated laser points manually, where detection failed. I tried to detect laser points for individual images by clicking on the "i" icon of an image in the volume overview. I get the same message, "The laser point detection was submitted and will be available soon," but nothing happens after that, i.e., when I check if the lasers are detected and the area is calculated, it says, "No laser point detection was performed yet."
In the second volume, I tried to perform the automatic detection of laser pointers, too. Very few lasers were detected, and some of them were detected incorrectly. Then, I annotated laser pointers manually for those images where automatic detected failed (over 800). Then, I tried to perform laser pointer detection (selected the label used to annotate lasers and entered the distance between them), but I get a message "Delphi laser point detection can't be performed. Only 0 images have manually annotated laser points. At least 4 are required."
What I might be doing wrong?

**Comment by mzur** [comment]
Can you please show me the link to your volume so I can have a look? You can also send it to info@biigle.de if you don't want to show it here. We observed a few error messages in our logs which were related to laser point detection but we didn't have the time to look into that.

**Reply by Izh003** [reply]
I sent the link to info@biigle.de. Thank you!
